python/paddle/distributed/passes/pass_utils.py

```diff
@@ -34,6 +36,14 @@
 ]
 
 
+# NOTE: Here stream is just a presentation with different name,
+# it is up to executor to create the exact streams given the name.
+class AutoParallelStreamType(Enum):
```

Comment:
data parallel allreduce stream (in data_parallel_pass) maybe out of control by this TYPE.